.test/run.sh

```diff
@@ -608,6 +608,53 @@ fi
 	;&
 	password_hash)
 
+	echo -e "Test: Replica container can be initialized with environment variables\n"
+
+	mariadb_replication_user='foo'
+	mariadb_replication_password='foo123'
+	
+	runandwait -e MARIADB_REPLICATION_MODE="master" \
+		-e MARIADB_REPLICATION_USER="$mariadb_replication_user" \
+		-e MARIADB_REPLICATION_PASSWORD="$mariadb_replication_password" \
+		-e MARIADB_DATABASE=repcheck \
+		"$image"
```

Comment:
when I attempt to run the test of a build version of 10.5 tagged ms105:
```
$ .test/run.sh ms105 password_hash
mariadb  Ver 15.1 Distrib 10.5.19-MariaDB, for debian-linux-gnu (x86_64) using readline 5.2
Test: Replica container can be initialized with environment variables

waiting to start...
+ '[' 9 -eq 0 ']'
++ mariadbclient -u foo -proger repcheck --batch --skip-column-names -e 'select sum(i) from t1'
++ docker exec -i mariadbcontainer17535 mariadb --host 127.0.0.1 --protocol tcp --silent -u foo -proger repcheck --batch --skip-column-names -e 'select sum(i) from t1'
Error: no container with name or ID "mariadbcontainer17535" found: no such container
+ count_primary=
+ killoff
+ '[' -n 616c3018fbbedab037c018106ecc1d4e8e829a5b95771c9d06ba3e5bd19bfb5b ']'
+ docker kill 616c3018fbbedab037c018106ecc1d4e8e829a5b95771c9d06ba3e5bd19bfb5b
Error: no container with name or ID "616c3018fbbedab037c018106ecc1d4e8e829a5b95771c9d06ba3e5bd19bfb5b" found: no such container
```

testing further with:
```
$ bash -x -v .test/run.sh ms105 password_hash
```

Shows the docker line that start the container to be:
```
$ docker run -d --name mariadbcontainer15397 --rm --publish 3306 -e MARIADB_REPLICATION_MODE=master -e MARIADB_REPLICATION_USER=foo -e MARIADB_REPLICATION_PASSWORD=foo123 -e MARIADB_DATABASE=repcheck ms105
```

Remove the `-d` to get a foreground test run:
```
$ docker run --name mariadbcontainer15397 --rm --publish 3306 -e MARIADB_REPLICATION_MODE=master -e MARIADB_REPLICATION_USER=foo -e MARIADB_REPLICATION_PASSWORD=foo123 -e MARIADB_DATABASE=repcheck ms105
2023-04-15 00:17:44+00:00 [Note] [Entrypoint]: Entrypoint script for MariaDB Server 1:10.5.19+maria~ubu2004 started.
2023-04-15 00:17:44+00:00 [Note] [Entrypoint]: Switching to dedicated user 'mysql'
2023-04-15 00:17:44+00:00 [Note] [Entrypoint]: Entrypoint script for MariaDB Server 1:10.5.19+maria~ubu2004 started.
2023-04-15 00:17:44+00:00 [ERROR] [Entrypoint]: Database is uninitialized and password option is not specified
	You need to specify one of MARIADB_ROOT_PASSWORD, MARIADB_ROOT_PASSWORD_HASH, MARIADB_ALLOW_EMPTY_ROOT_PASSWORD and MARIADB_RANDOM_ROOT_PASSWORD
```

So adding `-e MARIADB_ALLOW_EMPTY_ROOT_PASSWORD=1` allows it to start.

Reply:
Done 👍🏽